tests/FSharp.Data.GraphQL.IntegrationTests.Server/JsonConverters.fs

```diff
@@ -1,31 +1,182 @@
 ﻿namespace FSharp.Data.GraphQL.IntegrationTests.Server
```

Comment:
I suppose this file is also a copy from StarWars? All the files we can reuse we want to reuse